README.md

```diff
@@ -39,6 +40,16 @@ services:
         ENTRYPOINT: ./YOUR_ENTRYPOINT
 ```
 
+By default, the IOC will be built with statically linked EPICS libraries. If
+you **need** to link them dynamically, you must define the build target as
+`dynamic-link`. This will increase the resulting image size, since unused
+dependencies will also be copied.
+
+Enabling [BuildKit](https://docs.docker.com/build/buildkit/) (for instance, by
+exporting `DOCKER_BUILDKIT=1`) when building the IOC image can be necessary on
+some systems, because the classic builder goes through all stages even when
+they are not need and they fail.
```

Comment:
I think this wording is a bit confusing... I'd say something like:

```
Some Docker versions don't use [BuildKit](https://docs.docker.com/build/buildkit/) by
default, and it is necessary to enable it by exporting `DOCKER_BUILDKIT=1` when
building the IOC image, because the classic builder goes through all stages even
when they are not needed or are known to fail.
```

Reply:
I understand. I was willing to let it up to the user define how she want to enable BuildKit. It may also be enabled through Docker daemon's configuration file, which can be more practical. But I agree the wording can be improved. How about something like this?

> Some Docker versions don't use [BuildKit](https://docs.docker.com/build/buildkit/) by default, and it is necessary to enable it, for instance, by exporting `DOCKER_BUILDKIT=1` when building the IOC image, because the classic builder goes through all stages even when they are not needed or are known to fail.

Reply:
I hadn't thought about changing the daemon's default :) That's a good point.